wordpress-mobile/WordPress-Android
# The inline action of Like Comment

Part of #20024

This is the feature for liking a comment from the inline action.
This PR contains:
1. The implementation of the Like Comment inline action.
2. Refactor the codebase.

-----

## To Test:
1. Open the Jetpack app and switch to the notifications tab.
3. Scroll the top tab to COMMENTS and click it.
4. Click on the star icon in the notification item.
5. The Like status should be updated.
6. Done, thank you!

| Light | Dark | RTL |
|--|--|--|
| ![截圖 2024-02-15 下午2 39 16](https://github.com/wordpress-mobile/WordPress-Android/assets/3839951/cbfdf88f-8059-40ea-9218-62494b6f6b23) | ![截圖 2024-02-15 下午2 39 25](https://github.com/wordpress-mobile/WordPress-Android/assets/3839951/21692ec6-946b-43ea-bd74-2bbea47a3d86) | ![截圖 2024-02-15 下午3 04 42](https://github.com/wordpress-mobile/WordPress-Android/assets/3839951/b75ce717-428f-45bf-99b5-9da70837f3dd) |




-----

## Regression Notes

1. Potential unintended areas of impact

    - Notification tab

8. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual

9. What automated tests I added (or what prevented me from doing so)

    - None, sadly we still haven't made our code testable yet.

-----

## PR Submission Checklist:


-----

## Testing Checklist:
